File: C#PartOne/Homeworks/06.Loops/MMSA/MMSA.cs
namespace MMSA
{
    using System; 
    using System.Globalization;
    using System.Linq;
    using System.Threading;
  
    public class MMSA
    {
        public static void Main(string[] args)
        {
            Thread.CurrentThread.CurrentCulture = CultureInfo.InvariantCulture;

            int n = int.Parse(Console.ReadLine());
            double[] numbers = new double[n];

            for (int i = 0; i < n; i++)
            {
                numbers[i] = double.Parse(Console.ReadLine());
            }

            Array.Sort(numbers);
            double min = numbers[0];
            double max = numbers[n - 1];
            double sum = numbers.Sum();
            double average = numbers.Average();

            Console.WriteLine("min={0:F2}", min);
            Console.WriteLine("max={0:F2}", max);
            Console.WriteLine("sum={0:F2}", sum);
            Console.WriteLine("avg={0:F2}", average);
        }
    }
}

File: JavaScript-Fundamentals/Homeworks/Loops/02. MMSA/MMSA.js
function solve(args) {
    var numbers = [],
        min,
        max,
        sum = 0,
        average;

    for (var i = 0; i < args.length; i += 1) {
        numbers.push(+args[i]);
    }

    min = Math.min.apply(null,numbers);
    max = Math.max.apply(null,numbers);
    for (var j = 0; j < numbers.length; j += 1) {
        sum += numbers[j];
    }
    average = sum / numbers.length;

    console.log('min='+min.toFixed(2));
    console.log('max='+max.toFixed(2));
    console.log('sum='+sum.toFixed(2));
    console.log('avg='+average.toFixed(2));
}

var test = ['2','5','1'];
solve(test);